### PR TITLE
Completed javascript-light-bulb exercise.

### PR DESCRIPTION
**Pull request by danielycho**
light-bulb-2
light-bulb-1

### CODE REVIEW

File: javascript-light-bulb/main.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+var totalClick = 0;
+
+var light = document.querySelector('.circle');
+var background = document.querySelector('.container');
+
+function onAndOff(event) {
+  totalClick++;
+  if (totalClick % 2 !== 0) {
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:8]: You aren't using the click count here for anything other than checking if the value is even or odd. In this case, a variable holding a boolean value would be a better way of keeping track of if the light is currently on or not.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by danielycho** [comment]
image

File: javascript-light-bulb/main.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+var statusOfLight = true;
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:1]: A better name for this variable would be `isLightOn`. That makes the conditional statement below make more sense.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by danielycho** [comment]
image

Changed variable.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**